### ISSUE TITLE
Batch Normalization - Rui's Blog

### ISSUE THREAD

**Issue by SupeRuier** [SupeRuier/blog-comment#27]
https://blog.superui.cc/machine-learning/batch-normalization/
